### PR TITLE
Add numerous DevOpsDays conferences

### PR DESCRIPTION
**Pull request by phrawzty**
Add numerous DevOpsDays 🧠⚙️ conferences, and correct a handful that were already listed.

Source: https://devopsdays.org/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ All the data (past and coming) are available publicly in JSON:
 * 16: [DDD Melbourne](https://www.dddmelbourne.com/) - Melbourne (Australia)
 * 16: [Global AI Bootcamp 2024 - Mumbai](https://lu.ma/gaib24mum) - Mumbai (India) <a href="https://sessionize.com/gaib24mum/"><img alt="CFP Global AI Bootcamp 2024 - Mumbai" src="https://img.shields.io/static/v1?label=CFP&message=until%2012-March-2024&color=green"></a>
 * 17-18: [Cloud Native Rejekts EU 2024](https://cloud-native.rejekts.io/) - Paris (France)
+* 18-19: [DevOpsDdays Kraków 2024](https://devopsdays.org/events/2024-krakow/welcome/) - Kraków (Poland)
```
Comment (scraly) [inline README.md:80]:
```suggestion
* 18-19: [DevopsDays Kraków 2024](https://devopsdays.org/events/2024-krakow/welcome/) - Kraków (Poland)
```

File: README.md [hunk 2]
```diff
@@ -440,31 +449,35 @@ All the data (past and coming) are available publicly in JSON:
 
 * 3-4: [ContainerDays Conference 2024](https://www.containerdays.io/containerdays-conference-2024/) - Hamburg (Germany) <a href="https://sessionize.com/CDS-conference-2024/"><img alt="CFP ContainerDays Conference 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-March-2024&color=green"></a>
 * 4-5: [JavaZone](https://2024.javazone.no/) - Oslo (Norway) <a href="https://2024.javazone.no/speakers"><img alt="CFP JavaZone 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2008-April-2024&color=green"></a>
+* 4-5: [DevOpsDays Ghentwerp)](https://devopsdays.org/events/2024-ghent) - Antwerp (Belgium)
```
Comment (scraly) [inline README.md:452]:
```suggestion
* 4-5: [DevOpsDays Antwerp)](https://devopsdays.org/events/2024-ghent) - Antwerp (Belgium)
```

**Review by scraly — CHANGES_REQUESTED** [review]
Thanks!
just a few remarks :)

File: README.md (hunk 1 above)
Comment (phrawzty) [reply README.md:80]: `DevOpsDays` is the correct spelling.

Source: I am co-chair of the DevOpsDays conference series. :)

File: README.md (hunk 2 above)
Comment (phrawzty) [reply README.md:452]: The conference is called "DevOpsDays Ghent", but it's taking place in Antwerp, so our code-name for it is Ghentwerp. 😉 In fact, the "real name" for it is "DevOpsDays Ghent (Antwerp)".

Comment (scraly) [reply README.md:452]: ohh thanks for the information

File: README.md (hunk 1 above)
Comment (scraly) [reply README.md:80]: Nice to meet you :)
Yes I know the spelling but I saw a double d ^^

**Pull request merged**